.circleci/config.yml

```diff
@@ -501,7 +501,7 @@ jobs:
             python3 build.py -a --nocache --parallel
       - tag-push-docker:
           project: orc8r
-          images: "nginx|controller"
+          images: "nginx|controller|base"
```

Comment:
why did you add `base` ?

Reply:
I didn't add this base, don't know how come it showing as change